src/components/QuestDetailsWrapper/index.tsx

```diff
@@ -625,7 +629,7 @@ export function QuestDetailsWrapper({
         />
         <QuestDetails
           {...questDetailsProps}
-          className={styles.questDetails}
+          className={cn(questDetailsProps.className, styles.questDetails)}
```

Comment:
```suggestion
          className={cn(styles.questDetails, questDetailsProps.className)}
```

I prefer to put the prop className after so that we can override the default styling if needed